gcp/extract/lib/bundles.py

```diff
@@ -48,14 +48,11 @@ def read_region(target_regions, *args, **kwargs):
     regions : array-like
     data : array-like
     """
-    # Because someone's always tryin to ice-skate up hill
-    if isinstance(target_regions, str):
-        target_regions = [target_regions]
-
     years, regions, data = read(*args, **kwargs)
+    target_regions = configs.get_regions(config, regions)
     regions_msk = np.isin(regions, target_regions)
```

Comment:
Move this below the `configs.is_allregions` check, so we don't need to run an N^2 operation every time.

Reply:
Applied the recommended changes. Closing.